devicetypes/smartthings/everspring-outdoor-floodlight.src/everspring-outdoor-floodlight.groovy

```diff
@@ -0,0 +1,162 @@
+/**
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+metadata {
+	definition(name: "Everpsring Outdoor Floodlight", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.light", mmnm: "SmartThings", vid: "generic-motion-light-sensor") {
```

Comment:
Want to rename this something like "Z-Wave Motion Light"? Nothing in this seems particularly non-standard. It might even run locally.